modules_extra/notifications.js

```diff
@@ -106,15 +110,15 @@ exports.create = function (api) {
   }
 
   function getFirstMessage(feedId, cb) {
-    api.sbot_user_feed({id: feedId, gte: 0, limit: 1})(null, cb)
+    api.sbot.user_feed({id: feedId, gte: 0, limit: 1})(null, cb)
```

Comment:
now that we have proper namespacing through nesting, can we use `camelCase`?

@dominictarr curious, what was the initial reason for `snake_case` in patchbay?